package.json

```diff
@@ -0,0 +1,88 @@
+{
+  "name": "@unfinishedlabs/test-generators",
+  "version": "0.0.0",
+  "description": "Utilities to generate test data",
+  "type": "commonjs",
+  "main": "dist/cjs/index.js",
+  "types": "dist/types/index.d.ts",
+  "module": "dist/esm/index.js",
```

Comment:
These currently point to `index`, but we are only generating `generators`. We should keep this, but change `generators.ts` to `index.ts`.

Reply:
done.